lighthouse-core/scripts/internal-analysis/analyze-issues.js

```diff
@@ -0,0 +1,275 @@
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const path = require('path');
+const log = require('lighthouse-logger');
+
+/**
+ * @fileoverview Used in conjunction with `./download-issues.js` to analyze our Issue and PR response times as a team.
+ *
+ * This file analyzes GitHub data that resides in `.tmp/_issues.json` primarily around *initial* response times.
+ * Future work could do something fancier around responding to replies, followup reviews, closing issues, etc.
+ *
+ * See the download script for usage information.
+ */
+
+/** @typedef {import('./download-issues.js').AugmentedGitHubIssue} AugmentedGitHubIssue */
+
+const RESPONSE_LOGINS = new Set([
+  'adamraine',
+  'Beytoven',
+  'brendankenny',
+  'connorjclark',
+  'exterkamp',
+  'jazyan',
+  'patrickhulce',
+  'paulirish',
+]);
+
+/**
+ * @param {AugmentedGitHubIssue} issue
+ * @return {AugmentedGitHubIssue}
+ */
+function normalizeIssue(issue) {
+  if (!Array.isArray(issue.comments)) issue.comments = [];
+  if (!Array.isArray(issue.events)) issue.events = [];
+  issue.comments = issue.comments.sort(
+    (a, b) =>
+      new Date(a.created_at).getTime() - new Date(b.created_at).getTime()
+  );
+  issue.events = issue.events.sort(
+    (a, b) =>
+      new Date(a.created_at).getTime() - new Date(b.created_at).getTime()
+  );
+
+  issue.comments.forEach(comment => {
+    comment.created_at = comment.created_at || comment.submitted_at || '';
+  });
+
+  return issue;
+}
+
+const RESPONSE_EVENTS = new Set(['labeled', 'assigned', 'renamed', 'closed']);
+
+const HOUR_IN_MS = 60 * 60 * 1000;
+const DAY_FILTER = 90;
+const START_AT = new Date(
+  new Date().getTime() - DAY_FILTER * 24 * HOUR_IN_MS
+).getTime();
+
+const ISSUES_PATH = path.join(__dirname, '../../../.tmp', '_issues.json');
+/** @type {Array<AugmentedGitHubIssue>} */
+const _ISSUES = JSON.parse(fs.readFileSync(ISSUES_PATH, 'utf8')).map(
+  normalizeIssue
+);
+const _ISSUES_SINCE = _ISSUES.filter(
+  issue => new Date(issue.created_at).getTime() > START_AT
+);
+const ISSUES = _ISSUES_SINCE.filter(issue => !issue.pull_request);
+
+/** @param {number} n */
+const percent = n => `${(n * 100).toFixed(1)}%`;
+
+/**
+ * This function only logs *initial* review responses, but could conceivably log all instances of applying "waiting4reviewer".
+ * @param {string} label
+ * @param {Array<AugmentedGitHubIssue>} issues
+ */
+function computeAndLogReviewResponseStats(label, issues) {
+  const initialReviewRequests = issues
+    .map(issue => {
+      const assignEvent = issue.events.find(
+        event => event.event === 'assigned'
```

Comment:
what is this 80 character nonsense about? :)

Reply:
so apparently the existence of a `.prettierrc` in a subfolder nulls out VSCode prettier user settings, fixed 👍

Reply:
also can we please just move it out from `test/` and finally start allowing us to use prettier for new files, it's not like we have to enforce it :)

Reply:
https://twitter.com/drosenwasser/status/1255282740808859648 :)

Reply:
https://twitter.com/ferdaber/status/1255285797282025475 :)